eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -2,426 +2,458 @@
-        "new": "method reactor.core.publisher.Mono<java.io.InputStream> com.azure.core.http.HttpResponse::getBodyAsInputStream()",
-        "justification": "New default method added to HttpResponse in azure-core."
-      },
-      {
-        "code": "java.method.added",
-        "new": "method java.lang.String com.azure.core.credential.TokenRequestContext::getTenantId()",
-        "justification": "New method added to TokenRequestContext in azure-core."
-      },
-      {
-        "code": "java.method.added",
-        "new": "method com.azure.core.credential.TokenRequestContext com.azure.core.credential.TokenRequestContext::setTenantId(java.lang.String)",
-        "justification": "New method added to TokenRequestContext in azure-core."
-      },
-      {
-        "code": "java.method.added",
-        "new": "method java.lang.String com.azure.storage.common.StorageSharedKeyCredential::generateAuthorizationHeader(java.net.URL, java.lang.String, com.azure.core.http.HttpHeaders, boolean)",
-        "justification": "New method added to SharedKeyCredential in common not intended for use by customers. Only public for access by other packages."
-      },
-      {
-        "code": "java.method.added",
-        "new": "method <T, U> com.azure.core.util.polling.PollerFlux<T, U> com.azure.core.util.polling.PollerFlux<T, U>::create(java.time.Duration, java.util.function.Supplier<reactor.core.publisher.Mono<? extends com.azure.core.http.rest.Response<?>>>, com.azure.core.util.polling.PollingStrategy<T, U>, com.azure.core.util.serializer.TypeReference<T>, com.azure.core.util.serializer.TypeReference<U>)",
-        "justification": "New method added to PollerFlux in azure-core."
-      },
-      {
-        "code": "java.method.added",
-        "new": "method java.lang.Class<T> com.azure.core.util.serializer.TypeReference<T>::getJavaClass()",
-        "justification": "New method added to TypeReference in azure-core."
+    "extension": "revapi.filter",
+    "configuration": {
+      "elements": {
+        "exclude": [
+          "class com\\.azure\\.core\\.util\\.Configuration",
+          "class com\\.azure\\..+\\.implementation(\\.)?.+",
+          "class com\\.azure\\..+\\.samples(\\..+)?",
+          "class com\\.fasterxml\\.jackson\\..+",
+          "class reactor\\.core\\..+",
+          "class reactor\\.netty\\..+",
+          "class reactor\\.util\\..+",
+          "class io\\.netty\\..+",
+          "class com\\.nimbusds(\\..+)?",
+          "class com\\.microsoft\\.azure\\..+",
+          "class javax\\.jms(\\..+)?",
+          "class javax\\.servlet(\\..+)?",
+          "class io\\.micrometer(\\..+)?",
+          "class com\\.azure\\.data\\.cosmos(\\..+)?",
+          "class com\\.azure\\.spring\\.data\\.cosmos\\..+",
+          "class com\\.azure\\.spring\\.data\\.gremlin\\..+",
+          "class com\\.azure\\.spring(\\..+)?",
+          "class org\\.apache\\.avro\\..+",
+          "class com\\.azure\\.resourcemanager\\..*\\.fluent\\..*"
```

Comment:
Not all resourcemanager libraries exclude this package. For example, [applicationinsights](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/applicationinsights/azure-resourcemanager-applicationinsights/pom.xml) doesn't skip this.

Reply:
I wonder if that is due to the fact it has never GA'd?